src/manifests_workflow/input_manifests.py

```diff
@@ -110,8 +110,9 @@ def update(
             logging.info(f"Ignoring {self.name} {sorted(set(all_branches) - set(branches))} branches as they are legacy")
 
             for branch in branches:
+                repo_path = os.path.join(work_dir.name, self.name.replace(" ", ""), branch)
```

Comment:
Any reason we use a new variable for this?

Reply:
My IDE complains the line being too long, so I just change to a new var.
A bit more clear on the code as well.